back/app/services/community_monitor_service.rb

```diff
@@ -73,9 +81,59 @@ def create_and_set_project(project: nil, current_user: nil)
     project
   end
 
+  def find_or_create_previous_quarter_report
+    return nil unless enabled? && project_id
+
+    start_date, end_date = previous_quarter_range
+    year, quarter = previous_quarter(start_date)
+
+    # Return existing report if it exists
+    existing_report = ReportBuilder::Report.find_by(
+      phase: phase,
+      year: year,
+      quarter: quarter
+    )
+    return existing_report if existing_report
+
+    # Do not create a report if there are no responses for the previous quarter
+    responses = phase.ideas.where(published_at: start_date..end_date)
+    return nil if responses.blank?
```

Comment:
This also means the email campaign won't be sent whenever a report isn't created for a quarter right?